src/components/MediaPlayer/MediaPlayer.js

```diff
@@ -290,7 +290,7 @@ const MediaPlayer = ({
    * @param {String} focusElement element to be focused within the player when using
    * next or previous buttons with keyboard
    */
-  const switchPlayer = (index, fromStart, checkAutoAdvance = false, focusElement = '') => {
+  const switchPlayer = (index, fromStart, focusElement = '', checkAutoAdvance = false) => {
```

Comment:
Why did you switch these params?

Reply:
This was to experiment whether I could get the previous/next button focus retained in the player controls after using the previous/next buttons in the inaccessible message display, which I couldn't. This change doesn't make a difference I guess. But I can check again.